docs/elasticsearch/reindex-data.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Reindex Data
+parent: Elasticsearch
+nav_order: 3
+---
+
+# Reindex data
+
+After creating an index, if you need to make an extensive change such as adding a new field to every document or performing some computation on all integer fields, rather than deleting your index, making the change offline, and then indexing your data all over again, you can use the `_reindex` operation.
+
+With the `_reindex` operation, you can copy all or a subset of documents that you select through a query to another index. Reindex is a `POST` operation and you need to specify a source and destination index.
+
+Reindexing can be an expensive operation depending on the size of your source index. We recommend you disable all replicas on your destination index and re-enable them once the reindex process is complete.
+{: .note }
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+
+---
+
+## Reindex all documents
+
+You can copy all documents from one index to another.
+
+This command copies all the documents from a source index to a destination index:
+
+```json
+POST _reindex
+{
+   "source":{
+      "index":"source"
+   },
+   "dest":{
+      "index":"destination"
+   }
+}
+```
+
+The settings from the source index are not copied over, so if the destination index is not already created, the `_reindex` operation creates a new destination index with default configurations.
```

Comment:
I think it's worth telling people straight-up that they should create an index with their desired settings before running the reindex. Let's provide a sample call for index creation with non-default settings.